plutus-cbor/plutus-cbor.cabal

```diff
@@ -85,7 +85,7 @@ test-suite unit
   hs-source-dirs:     test
   main-is:            Main.hs
   ghc-options:        -threaded -rtsopts
-  build-tool-depends: hspec-discover:hspec-discover -any
```

Comment:
what is this -any argument for and why are we removing this?

Reply:
It's a version indicator I guess. My formatter is removing it, and other peoples formatter is adding it .. or the other way around. It doesn't change anything if we have it or not.